osh/hub/osh_xmlrpc/worker.py

```diff
@@ -140,6 +147,9 @@ def open_task(request, task_id):
             task.worker.max_load = 0
             task.worker.save()
 
+    msg = message.Message(topic=u'task.open', headers={u'task_id': task_id}, body={})
+    api.publish(msg)
```

Comment:
Can we interpret this as the task is now "in-progress"?

Reply:
Yes, when the task is in 'open' stage, a build is being performed by the worker.